.github/workflows/automatic-testing.yml

```diff
@@ -45,12 +45,20 @@ jobs:
       - name: Run minitwit
         run: |
           docker compose -f docker-compose-tests.yaml build
-          # https://stackoverflow.com/questions/40907954/terminate-docker-compose-when-test-container-finishes
           yes 2>/dev/null | docker compose -f docker-compose-tests.yaml up --abort-on-container-exit --exit-code-from test-runner
         env:
           DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
           MINITWIT_VERSION: ${{ github.sha }}
 
+      - name: Run e2e minitwit
+        run: |
+          docker compose -f docker-compose-e2e-test.yaml build
+          yes 2>/dev/null | docker compose -f docker-compose-e2e-test.yaml up --abort-on-container-exit --exit-code-from e2e-test-runner
+        env:
+          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+          MINITWIT_VERSION: ${{ github.sha }}
+
+
```

Comment:
Should we not upload the generated report on failed/all runs?

Something like:
```
    - uses: actions/upload-artifact@v4
      if: ${{ !cancelled() }}
      with:
        name: playwright-report
        path: playwright-report/
        retention-days: 30
```

However requires that we also make sure playwright.config.ts generates and puts the report to `./playwright-report/.` i suppose.